scss/_forms.scss

```diff
@@ -142,7 +128,5 @@ input[type="radio"] {
 }
 
 select[multiple] {
-  margin-bottom: 0;
```

Comment:
@tofran this causes inconsistency in select margin?
Can you show examples with and without this rule?